### DISCUSSION TITLE
DuckDB

### DISCUSSION THREAD

**Discussion by ulissescappato** [Mimetis/Dotmim.Sync#950] [Q&A]
Hi!

Any plans to work with DuckDB database? It is similar to SQLite, but for analytics.

Thanks,
Ulisses.

**Comment by Mimetis** [comment]
No plan for now, honestly.
If it's similar to **SQLite**, I think you can try to create your own **Sync Provider** based on [SqliteSyncProvider](https://github.com/Mimetis/Dotmim.Sync/blob/master/Projects/Dotmim.Sync.Sqlite/SqliteSyncProvider.cs)

And maybe create a PR for it ? :)

As it's not a DMS issue, and more a question, I'll move the issue to the discussion tab.
